### PR TITLE
fix: azure file latency issue for metadata-heavy workloads

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
fix: azure file latency issue for metadata-heavy workloads

details: https://docs.microsoft.com/en-us/azure/storage/files/storage-troubleshooting-files-performance#high-latencies-for-metadata-heavy-workloads-involving-extensive-openclose-operations

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
fix: azure file latency issue for metadata-heavy workloads
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
fix: azure file latency issue for metadata-heavy workload
```

/kind bug
/priority important-soon
/sig cloud-provider
/area provider/azure
/triage accepted

### CODE REVIEW

File: pkg/volume/azure_file/azure_util.go [hunk 1]
```diff
@@ -138,5 +149,13 @@ func appendDefaultMountOptions(mountOptions []string, fsGroup *int64) []string {
 	if !gidFlag && fsGroup != nil {
 		allMountOptions = append(allMountOptions, fmt.Sprintf("%s=%d", gid, *fsGroup))
 	}
+
+	if !actimeoFlag {
+		allMountOptions = append(allMountOptions, fmt.Sprintf("%s=%s", actimeo, defaultActimeo))
+	}
+
+	if !mfsymlinksFlag {
```
Comment (feiskyer) [inline pkg/volume/azure_file/azure_util.go:157]: should it be `if mfsymlinksFlag` here?

File: pkg/volume/azure_file/azure_util.go [hunk 2]
```diff
@@ -138,5 +149,13 @@ func appendDefaultMountOptions(mountOptions []string, fsGroup *int64) []string {
 	if !gidFlag && fsGroup != nil {
 		allMountOptions = append(allMountOptions, fmt.Sprintf("%s=%d", gid, *fsGroup))
 	}
+
+	if !actimeoFlag {
```
Comment (feiskyer) [inline pkg/volume/azure_file/azure_util.go:153]: should it be `if actimeoFlag` here?

File: pkg/volume/azure_file/azure_util.go (hunk 1 above)
Comment (andyzhangx) [reply pkg/volume/azure_file/azure_util.go:157]: `if !mfsymlinksFlag` means user did not set that flag, then azure file driver should append that mount option.

**Review by ZeroMagic — APPROVED** [review]
/lgtm

**Pull request merged**